packages/content/types/parsers/csv/index.d.ts

```diff
@@ -0,0 +1,10 @@
+export class ICSV {
```

Comment:
options can be typed

Reply:
Updated it to a class with correct options according to documenation